listener/pkg/event/skr_events_listener.go

```diff
@@ -82,12 +80,9 @@ func (l *SKREventListener) Start(ctx context.Context) error {
 
 func (l *SKREventListener) RequestSizeLimitingMiddleware(next http.HandlerFunc) http.HandlerFunc {
 	return func(writer http.ResponseWriter, request *http.Request) {
-		_, err := io.ReadAll(request.Body)
 
-		if request.ContentLength > requestSizeLimitInBytes ||
-			errors.Is(err, &http.MaxBytesError{Limit: requestSizeLimitInBytes}) {
+		if request.ContentLength > requestSizeLimitInBytes {
 			errorMessage := fmt.Sprintf("Body size greater than %d bytes is not allowed", requestSizeLimitInBytes)
-			l.Logger.Error(err, errorMessage)
```

Comment:
Why was this logging removed?

Reply:
Because there is not error anymore, the `err` was initialised when reading from body, but since there is not reading, there is nor error :D

Reply:
Will remove old error, but keep logging :D Thanks for catching